services/cad-service/grpc_server.py

```
@@ -0,0 +1,93 @@
+        async for chunk in _async_iterate(export_solid_bytes(solid, export_format)):
+            yield pb2.ExportChunk(data=chunk)
+
+        context.set_trailing_metadata(
+            (
+                ("cad-export-format", descriptor.cadquery_format.lower()),
+                ("cad-export-extension", descriptor.extension),
+            )
+        )
+
+
+def _async_iterate(iterator):
```

Ruff (0.14.1)

Missing return type annotation for private function `_async_iterate`

(ANN202)